python-threatexchange/threatexchange/exchanges/impl/fb_threatexchange_api.py

```diff
@@ -148,15 +148,19 @@ def from_threatexchange_json(
 
             for reaction in td_json.get("reactions", []):
                 rxn = reaction["key"]
-                owner = int(reaction["value"])
-                if rxn == "HELPFUL":
-                    implicit_opinions[
-                        owner
-                    ] = state.SignalOpinionCategory.POSITIVE_CLASS
-                elif rxn == "DISAGREE_WITH_TAGS" and owner not in implicit_opinions:
-                    implicit_opinions[
-                        owner
-                    ] = state.SignalOpinionCategory.NEGATIVE_CLASS
+                for owner in reaction["value"].split(","):
```

Comment:
Can you add this link somewhere? 

It gets hard to understand the diff and the code without knowing what the JSON structure is.

https://developers.facebook.com/docs/threat-exchange/reference/apis/reaction-type/v14.0